### DISCUSSION TITLE
iPad app

### DISCUSSION THREAD

**Discussion by Abod1960** [toeverything/AFFiNE#7583] [Ideas]
I'm sure you have this in mind but thought about adding more voices, I hope you will provide an iPad app soon, that would make a great advancement for the app in terms of reach and functionality. 
Thanks in advance.

**Comment by randall11GB** [comment]
adding Mobile is stated that is set for winter 2024.  I am using canary, on docker, selfhosted, behind Nginx proxy manager.  until few days ago i could view and edit using brave on android.  now the app says view only on mobile.
solution i am using is to set brave on android to desktop and can view and edit.  not ideal but better that not using.  
love app but mobile is a urgent MUST!

**Comment by yebei199** [comment]
apple pencil, start up

**Comment by gsgill112** [comment]
Any timelines for iPad app, as I see the iPad app in conjunction with Apple Pencil will be able to fully harness the power of the app, since mac books still don’t have a touch screen and no one is drawing using Mouse :P 

Also if there is a beta/ dev channel where we can contribute to the iPad app, with whatsoever little knowledge we have. :)

**Comment by Qenjin** [comment]
IOS and iPad app! Definitely looking to see where progress is on this.
